### DISCUSSION TITLE
How to generate document id?

### DISCUSSION THREAD

**Discussion by dongumayagay** [tylim88/FirelordJS#106] [Q&A] [answered]
in firebase sdk its

const db = firebase.firestore();
const ref = db.collection('your_collection_name').doc();

how about in firelord?

**Comment by tylim88** [comment, marked as answer]
> in firebase sdk its
> 
> const db = firebase.firestore();
> const ref = db.collection('your_collection_name').doc();
> 
> how about in firelord?

const doc = example.doc(example.collection())

const id = doc.id
